frontend/src/views/web/partials/dashboard-footer.html

```diff
@@ -6,7 +6,7 @@ <h5 class="white-text">EvalAI</h5>
                 <p class="text-light-gray">Evaluating state of the art in AI</p>
                 <ul class="inline-list">
                     <li><a class="text-light-gray" href="https://github.com/Cloud-CV/EvalAI" target="_blank"><i class="fa fa-github"></i></a></li>
-                    <li><a class="text-light-gray" href="https://twitter.com/project_cloudcv" target="_blank"><i class="fa fa-twitter"></i></a></li>
+                    <li><a class="text-light-gray twitter-share-button" href="https://twitter.com/intent/tweet?text=EvalAI%20is%20so%20Awesome http://evalai.cloudcv.org/" target="_blank"><i class="fa fa-twitter"></i></a></li>
```

Comment:
Hey, you have encoded spaces in the tweet text but left few spaces un-encoded. We can encode the rest of the URL too
```suggestion
                    <li><a class="text-light-gray twitter-share-button" href="https://twitter.com/intent/tweet?text=EvalAI%20is%20so%20Awesome%20http://evalai.cloudcv.org/" target="_blank"><i class="fa fa-twitter"></i></a></li>
```

Reply:
Thanks for the review @galipremsagar , incoporated the suggestions mentioned by you